functions-common/fetchFile.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const gitOpsApi = require("git-ops-api");
+
+/**
+ * Check if an aswer file in the given branch exists. If exists, along with the existence flag, return
+ * `content` and `sha` of this file that will be used at the file update.
+ * @param {Object} octokit - An instance of octokit with authentication being set.
+ * @param {String} branch - The name of the branch that the file existence will be checked against.
+ * @param {String} filePath - The file path.
+ * @return {Promise} The resolved promise contains an object with a boolean flag keyed by `exists`. If
```

Comment:
Slightly nicer to supply a typedef for this returned type and then write the JSDoc as @return {Promise<myType>}